guide/glossary/index.md

```diff
@@ -409,11 +409,12 @@ A transaction is a transfer of value over the bitcoin network. While transaction
 An output that has not been sent to another address. The bitcoin wallet balance is calculated from adding up unspent outputs.
 
 ###  Partially signed bitcoin transaction (PSBT)
-A file format for bitcoin transactions that are not fully signed yet. Allows for passing around a transaction to other applications or devices for signing, for example in a multi-signature wallet setup.
+A portable transaction data format that enables discrete participants to contribute to a single transaction. For example, psbt fascilitates contribution from both software and hardware wallet in multi-signature and between the sender and recipient of a payjoin.
```

Comment:
I think this misses the point about the data/file itself is referred to as the PSBT, which is possibly the dominant usage.

However, if I had to improve this suggestion, here it is:
```suggestion
A portable transaction data format that enables multiple participants to contribute to a single transaction. For example, PSBT facilitates contribution from both software and hardware wallet in a multi-signature setup and between the sender and recipient of a payjoin.
```

Reply:
Can you explain more? What else would it refer to? How can we lean more on the T for Transaction in PSBT? What else do people think PSBT refers to?

Reply:
I'd also like to make a suggestion. The sentence goes right away into use cases. How about first stating what it is - a data format for transactions before they are broadcast. Often, the same application constructs the application and signs it. But there are plenty of situations when this is not the case, like when using an external signer or in a multi-key scenario. That's where it's helpful when the transaction data can be passed around between applications.

Reply:
I like your suggestion about starting with a concrete *what*.

Instead of "being broadcast," a condition any data blob including psbt can fulfill, perhaps "being [consensus] verified [by network participants]." Originally I wanted to say "joining a mempool" because that term is defined in the glossary, but the definition doesn't mention that only consensus valid txs get into mempools.

> A portable data format for transactions before they are verified by the network. Often, the same application constructs the transaction and signs it. But there are plenty of situations when this is not the case, like when using a [hardware wallet](https://bitcoin.design/guide/glossary/#wallet), [multisig](https://bitcoin.design/guide/glossary/#multi-signature-wallet-multisig) or [payjoin](https://bitcoin.design/guide/glossary/#payjoin-p2ep). That's where it's helpful for PSBT transaction data to be passed between applications.

It's key to mention payjoin or another interactive tx construction ike dual-funded lightning to address the original goal of this issue, that PSBT isn't only for multisig (PSBTv0) but also for inputs from multiple participants (PSBTv2).

Reply:
That makes sense. Do you want to mention the two versions as well, or link out to another resource where users can dive deeper?

Reply:
Tried to incorporate the feedback so far.

```suggestion
When multiple signers are involved in creating a transaction, they have to pass around partially signed versions of the same between each other.  Partially signed bitcoin transaction refers to these and the standardised, portable data format (called the [PSBT](https://github.com/bitcoin/bips/blob/master/bip-0174.mediawiki) format) that enables them.

For example, PSBTs facilitate contributions from both software and hardware wallet in multi-signature (PSBT v0) and between the sender and recipient of a payjoin (PSBT v2).
```